### DISCUSSION TITLE
Help with Publishing and Deploying Serenity Project on IIS

### DISCUSSION THREAD

**Discussion by mayankkshah** [serenity-is/Serenity#7259] [Help]
I have been working on a Serenity project and now I'm trying to publish and deploy it using IIS. I've followed the basic steps for setting up IIS, but I'm facing some challenges with publishing the project and configuring the deployment.

Here are some details of my setup:
Project Type: Serenity Framework
Database: SQL Server
Operating System: Windows 11

Steps I've Tried So Far:
1. Published the project using Visual Studio’s "Publish" feature.
2. Tried to configure IIS with a new website and application pool, but ran into some issues.
Any help or guidance would be greatly appreciated!

Thank you!

**Comment by dfaruque** [comment]
Publishing a .net web application and a serenity application is the same process.
May be the correct .net version is not setup/registered in the IIS.

Exactly what trouble are you facing?

**Reply by mayankkshah** [reply]
I have Published the project using Visual Studio’s "Publish" feature the target is the Folder option.
<img width="585" alt="target" src="https://github.com/user-attachments/assets/33f4ba39-9f32-4cf4-a7e8-4d6f7ac8967e">
Then created a new website in the IIS. 
While browsing the website I am seeing all the files instead of the welcome page/Login page.
<img width="960" alt="Main" src="https://github.com/user-attachments/assets/b5f81417-272d-4838-8c80-3f8ce7a0e58b">
